src/components/BPageNav/BPageNav.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <nav>
+      <ul role="menu" aria-label="Page Navigation">
+        <li v-for="(link, index) in links" role ="menuitem" :key="index">
+          <a
+            :href="`#${link.href}`"
+            :aria-label="`${link.label} ${link.completed ? 'Completed' : 'Uncompleted'}`"
+            @click.prevent.stop="jumpTo(link)"
+            @keyup.enter="jumpTo(link)"
+            @keyup.space="jumpTo(link)"
+          >
+            <i v-if="link.completed" class="icon-complete ion-ios-checkmark" aria-hidden="true" />
+            <i v-else class="icon-incomplete ion-record" aria-hidden="true" />
+            {{ link.label }}
+          </a>
+        </li>
+      </ul>
+    </nav>
+</template>
+
+<script>
+import events from '../../event-bus'
+
+export default {
+  name: 'b-page-nav',
+  props: {
+    //Required props
+    links: {
+      type: Array,
+      required: true,
+      validator (value) {
+        for (let link of value) {
+          if (!link.label || !link.href)
+            throw new TypeError('Each link must have a label and href defined')
+        }
+        return true
+      }
+    }
+  },
+  data () {
+    return {
+    }
+  },
+  methods: {
+    jumpTo (link) {
+      if (link.event) {
+        events.$emit(link.event)
+      }
+      this.$nextTick(function () {
+        const element = document.getElementById(link.href)
+        if (element) {
+          element.scrollIntoView()
+          element.focus()
```

Comment:
Didn't see this in AC. Is this just for accessibility?

![screen shot 2018-02-09 at 9 06 17 am](https://user-images.githubusercontent.com/18075411/36036970-97471e34-0d78-11e8-8ac9-7ac1197d3b80.png)

Reply:
Yes, `element.focus` is for accessibility.